src/components/Weather/WeatherTile/styles.scss

```diff
@@ -0,0 +1,37 @@
+@import '../../../assets/styles/import.scss';
+@import '../../../variables.scss';
+
+.weather-tile {
+    display: flex;
+    padding: 1rem;
+    justify-content: space-between;
+    align-items: center;
+
+    &__weather-data {
+        align-items: center;
+        display: flex;
+        font-size: $font-sizes-extra-large4;
+
+        svg {
+            margin-right: 0.2rem;
+        }
+
+        &--color-red {
+            color: #ff5959;
```

Comment:
Er dette en entur-token? Isåfall kan det være smidig å bruke variabel fra @entur/tokens

Reply:
Det var det, fant den ikke først, men fant den nå!